assets/blocks/src/style.scss

```diff
@@ -0,0 +1,22 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+.wp-block-tour-block-tour-block {
+	background-color: #fff;
+	color: #000;
+	padding: 2px;
+}
+
+ul.tour-block {
+	//list-style: none;
+	//padding: 0 0 0 1rem;
```

Comment:
Can we remove this, since it's commented out already?

Reply:
Done. Thank you!